internal/operators/manager_test.go

```diff
@@ -76,34 +76,46 @@ var _ = Describe("Operators manager", func() {
 		})
 
 		It("should create 10 manifests (OCS + LSO) using the manifest API when OCS operator is enabled", func() {
-			cluster.MonitoredOperators = []*models.MonitoredOperator{
+			operators := []*models.MonitoredOperator{
 				&ocs.Operator,
 			}
 
+			resolvedOpeartors, err := manager.ResolveDependencies(operators)
```

Comment:
I think both LSO and OCS should be provided in the operators slice - the test is supposed to verify manifests generation and not dependencies resolution.